HW02/Tatsiana_Zenkovich/tatsiana_zenkovich_hw_02_t_01.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+lines = []
+File.open('log.txt') do |review_file|
+  lines = review_file.readlines
+end
+relevant_line = []
+lines.each do |line|
+  relevant_line << line if line.include?('error')
+end
```

Comment:
You can use `.select` here https://apidock.com/ruby/Array/select
```suggestion
lines.select { ... } 
```

Reply:
I corrected both, as you suggested

Reply:
Thank you!